### ISSUE TITLE
MySQL - mvn liquibase:generateChangeLog attempts to give JSON field definition a size

### ISSUE THREAD

**Issue by mkeathley** [liquibase/liquibase#3083]
## Environment

**Liquibase Version**: 4.13.0

**Liquibase Integration & Version**: maven

**Liquibase Extension(s) & Version**: n/a

**Database Vendor & Version**: MySQL v8.0.27

**Operating System Type & Version**: macOS Monterrey 12.4

**Infrastructure Type/Provider**: local

## Description

When generating a changelog for a table definition containing a field of type `JSON`, liquibase gives the field a max size even though MySQL does not have any such feature.  https://dev.mysql.com/doc/refman/8.0/en/json.html

## Steps To Reproduce

1. Create some test db using the creation script found in Additional Context.
2. Perform `mvn liquibase:generateChangeLog`.
3. Drop the table from the database.
4. Perform `mvn liquibase:update`.
5. You should get the error message `You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '(1073741824) NULL, CONSTRAINT PK_FOO PRIMARY KEY (id))' at line 1`

## Actual Behavior
In the changelog, the column definition is `<column name="some_json" type="JSON(1073741824)"/>`

## Expected/Desired Behavior
The column definition in the changelog should be `<column name="some_json" type="JSON"/>`

## Additional Context
DDL used to test:
```sql
create database test;

use test;

create table foo
(
    id        binary(16) default (uuid_to_bin(uuid(), true)) not null primary key,
    some_json json                                           null

);
```

pom.xml
```xml
<?xml version="1.0" encoding="UTF-8"?>

<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <modelVersion>4.0.0</modelVersion>

  <groupId>com.dialexa.mikekeathley</groupId>
  <artifactId>liquibase-defects-example</artifactId>
  <version>1.0-SNAPSHOT</version>

  <name>liquibase-defects-example</name>
  <!-- FIXME change it to the project's website -->
  <url>http://www.example.com</url>

  <properties>
    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    <maven.compiler.source>11</maven.compiler.source>
    <maven.compiler.target>11</maven.compiler.target>
  </properties>

  <dependencies>
    <dependency>
      <groupId>mysql</groupId>
      <artifactId>mysql-connector-java</artifactId>
      <version>8.0.27</version>
    </dependency>
  </dependencies>

  <build>
      <plugins>
        <!-- clean lifecycle, see https://maven.apache.org/ref/current/maven-core/lifecycles.html#clean_Lifecycle -->
        <plugin>
          <artifactId>maven-clean-plugin</artifactId>
          <version>3.1.0</version>
        </plugin>
        <!-- default lifecycle, jar packaging: see https://maven.apache.org/ref/current/maven-core/default-bindings.html#Plugin_bindings_for_jar_packaging -->
        <plugin>
          <artifactId>maven-resources-plugin</artifactId>
          <version>3.0.2</version>
        </plugin>
        <plugin>
          <artifactId>maven-compiler-plugin</artifactId>
          <version>3.8.0</version>
        </plugin>
        <plugin>
          <artifactId>maven-surefire-plugin</artifactId>
          <version>2.22.1</version>
        </plugin>
        <plugin>
          <artifactId>maven-jar-plugin</artifactId>
          <version>3.0.2</version>
        </plugin>
        <plugin>
          <artifactId>maven-install-plugin</artifactId>
          <version>2.5.2</version>
        </plugin>
        <plugin>
          <artifactId>maven-deploy-plugin</artifactId>
          <version>2.8.2</version>
        </plugin>
        <!-- site lifecycle, see https://maven.apache.org/ref/current/maven-core/lifecycles.html#site_Lifecycle -->
        <plugin>
          <artifactId>maven-site-plugin</artifactId>
          <version>3.7.1</version>
        </plugin>
        <plugin>
          <groupId>org.apache.maven.plugins</groupId>
          <artifactId>maven-dependency-plugin</artifactId>
          <version>3.2.0</version>
        </plugin>
        <plugin>
          <groupId>org.liquibase</groupId>
          <artifactId>liquibase-maven-plugin</artifactId>
          <version>4.13.0</version>
          <configuration>
            <propertyFile>liquibase.properties</propertyFile>
            <propertyFileWillOverride>true</propertyFileWillOverride>
          </configuration>
        </plugin>
      </plugins>
  </build>
</project>
```

liquibase.properties
```
driver=com.mysql.cj.jdbc.Driver
url=jdbc:mysql://localhost:3306
username=root
password=password
outputChangeLogFile=changelog-master.xml
changeLogFile=changelog-master.xml
defaultSchemaName=test
```

**Comment by FBurguer** [comment]
Duplicate of [#2208. ](https://github.com/liquibase/liquibase/issues/2208)

**Comment by MalloD12** [comment]
Fixed by #3849
